app/core/config.py

```diff
@@ -91,6 +100,24 @@ def compute_instructor_appstore_api_url(cls, v: Optional[str], values: Dict[str,
         if isinstance(v, str): return v
         return values.get("INSTRUCTOR_APPSTORE_HOST") + "/api/v1"
 
+    @validator("CELERY_BROKER_URI", pre=True)
+    def assemble_broker_uri(cls, v: Optional[str], values: Dict[str, Any]) -> Any:
```

Comment:
Shouldn't this one and the function below it return a `str`?

Reply:
I guess; it's inconsequential